### DISCUSSION TITLE
Is there a way to disable z-normalization?

### DISCUSSION THREAD

**Discussion by moriartyj** [TDAmeritrade/stumpy#228] [Q&A]
If I wanted to disable z_norm for my time series, is it just as simple as changing [this](https://github.com/TDAmeritrade/stumpy/blob/1c81ba3032a0dd9802f9d04f1ceef6a5e7a9025e/stumpy/floss.py#L452)?

**Comment by seanlaw** [comment]
@moriartyj Is this specifically for calling `floss`? I haven't thought about this in detail so please perform these changes at your own risk!

1. Yes, you'll need to remove the z-normalization in the initial step
2. In the update step, you'll want to replace the `mass` function call (which computes the distance profile for the new incoming subsequence) with another sliding window Euclidean distance computation

Again, know that this is not a supported implementation and I'm sure that I'm missing something

**Comment by seanlaw** [comment]
@moriartyj I'm going to close this for now but feel free to re-open when you are able to provide more details or if you have further questions. Thank you
